src/main.rs

```diff
@@ -218,11 +226,32 @@ impl Display for AllowanceJson {
     }
 }
 
+async fn init_senders_infinite_allowance_fastauth() {
```

Comment:
The relayer needs to know what type of service is calling it as it affects the specific functionality it will use based on the config. I included fastauth in the fn naming as it is only used for fastauth.